src/components/cards/Cards.jsx

```diff
@@ -1 +1,25 @@
 import "./css/cardsStyle.css";
+
+export function SuggestionCard(props) {
+  return (
```

Comment:
Dude, I agree with you that here he can destructure the props but when we have so many properties to our component and when we are adding a new prop whenever we want so at that time we are needed to mention that again while destructuring so we have to write that property name at both the places on the custom component and when we are using it so instead of that with just props we can access it without writing that term twice. I think about this destructuring the props thing that way.